### ISSUE TITLE
Agents seem (in my mind) to be too deterministic

### ISSUE THREAD

**Issue by Razvan00Rusu** [Razvan00Rusu/SOMAS2022-Team6#21]
idk if this will actually be the case or not - but it seems to me that bc every agent if following set formulas - they will all arrive to the same conclusions about everything.

Maybe we should inject some random numbers for some of the initial params or something to stop this from happening (if this does happen - need to test later)

**Comment by thl19git** [comment]
Yeah I would tend to agree. There will obviously be some differences depending on who trades with who, what items agents get etc., but particularly early on in the game I would expect the agents to all come to the same conclusions for the most part.

I like your suggestion about adding some non-determinism to the initial parameters. We can definitely add non-determinism to the initial manifesto, and we could also add it to the initial social capital values (with the exception of bravery).

**Comment by Issa-Bqain** [comment]
This is actually a feature. He covers this in the notes. This introduces the idea of populism and the collective intelligence idea tries to tackle this by  balancing it out with the use of 'expertise'.  @thl19git @Razvan00Rusu thoughts?

**Comment by thl19git** [comment]
> This is actually a feature. He covers this in the notes. This introduces the idea of populism and the collective intelligence idea tries to tackle this by balancing it out with the use of 'expertise'. @thl19git @Razvan00Rusu thoughts?

Features are good - which lecture was this in?

**Comment by Issa-Bqain** [comment]
I believe last one on democracies and governance
